lib/tool_belt/commands/check_deprecation_warnings.rb

```diff
@@ -0,0 +1,64 @@
+module ToolBelt
+  module Command
+    class CheckDeprecationWarningsCommand < Clamp::Command
+      parameter 'repo_path', 'Path (relative to tool_belt) of repository to check'
+      option '--version', 'VERSION', 'Release version to check for in x.y format', required: true
+      option '--paths', 'PATHS',
+             'Patterns within the repository to search, separated by a comma.',
+             default: 'app/**/*.rb,lib/**/*.rb'
+
+      def execute
+        root_path = File.expand_path("../../../../#{repo_path}", __FILE__)
+        globs_to_scan = paths.split(',').map { |path| File.expand_path(path, root_path) }
+        puts "\nScanning for deprecation warnings in #{globs_to_scan.join(', ')}\n\n"
+        paths_to_scan = Dir.glob(globs_to_scan)
+
+        finder = ToolBelt::OutdatedDeprecationWarningFinder.new(paths_to_scan, version)
```

Comment:
nit: is 'Outdated' really clarifying the purpose of the class? I'm thinking the name is a tad long (sorry, I said 'nit' :laughing: : )

Reply:
I know its probably invoking some Java flashbacks :smile: but I found "DeprecationWarningFinder" and "DeprecationWarningParser" too similar so tried to distinguish that one finds all deprecation warnings by parsing the code and the other gets you outdated deprecation warnings based on a version. I can brainstorm some other names

Reply:
No big deal if nothing comes to mind. I don't have a better idea either